File: code/src/test/scala/cern/colt/matrix/tlong/impl/SparseLongMatrix2DViewTest.scala
package cern.colt.matrix.tlong.impl

//remove if not needed
import scala.collection.JavaConversions._

class SparseLongMatrix2DViewTest(arg0: String) extends SparseLongMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new SparseLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new SparseLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new SparseLongMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tlong/impl/SparseLongMatrix2DViewTest.java
package cernx.colt.matrix.tlong.impl;

public class SparseLongMatrix2DViewTest extends SparseLongMatrix2DTest {

    public SparseLongMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseLongMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }
}
